bluej/src/main/java/bluej/pkgmgr/target/Target.java

```diff
@@ -304,6 +304,11 @@ else if (isMoveable() && !e.isStillSincePress())
 
                 e.consume();
             }
+            else if (Character.isJavaIdentifierStart(e.getCharacter().codePointAt(0)) || Character.isJavaIdentifierPart(e.getCharacter().codePointAt(0)))
```

Comment:
I wanted to ignore characters that aren't identifiers, because for example + and - change sizes in the interface, and space triggers the context menu, so we don't want to count them as part of a search.  I'll add a comment to this effect.